tests/popup-test.js

```diff
@@ -13,10 +13,71 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+require("../readiness-tool/popup");
 
-describe('test', function() {
+var chai = require('chai');
 
-   it('should pass', function() {
-     expect(1).to.equal(1);
-   });
- });
+var vendors = require("../readiness-tool/vendors");
+
+describe('isSupported(key)', function () {
+
+  it('should return true when the key given is in the supported list', function () {
+    expect(self.popups.isSupported('Adhese')).to.be.true;
```

Comment:
You can use Node modules instead of ES6 modules to avoid this. Remember that's how we fixed the early testing bug months ago? 

```js
// popup.js
module.exports = {isSupported, ...};

// popup-test.js
const art = require('path/to/popup');

expect(art.isSupported('Adhese')).to.be.true;
```

http://javascript.tutorialhorizon.com/2014/09/01/understanding-nodejs-module-exports-and-require/

Also, what version of Chrome are you running? ES6 modules should work in Chrome 60.